react-native/README.md

```diff
@@ -39,6 +39,7 @@ Download it [here.](./docs/atom-snippets.cson)
 
 ## Posts
 
+- [Introducing redux-recompose](https://medium.com/wolox-driving-innovation/932e746b0198)
```

Comment:
https://medium.com/wolox-driving-innovation/easy-data-management-from-a-rest-api-using-redux-recompose-v2-7c4dc5323445
https://medium.com/wolox-driving-innovation/fighting-back-fire-with-firebase-dc3e47021a81
https://medium.com/wolox-driving-innovation/the-importance-of-separating-structure-from-style-fa349e230b51

Reply:
the last one shouldn't be added since RN doesn't use html and css